core/src/main/resources/org/apache/spark/ui/static/executorspage-template.html

```diff
@@ -86,6 +86,22 @@ <h4 class="title-table">Executors</h4>
             <span data-toggle="tooltip" data-placement="top"
                   title="Memory used / total available memory for off heap storage of data like RDD partitions cached in memory.">
               Off Heap Storage Memory</span></th>
+          <th>
+            <span data-toggle="tooltip" data-placement="top"
+                  title="Peak onHeap / OffHeap memory used by JVM.">
+              Peak JVM Memory OnHeap / OffHeap</span></th>
+          <th>
+            <span data-toggle="tooltip" data-placement="top"
+                  title="Peak execution onHeap / OffHeap memory used by JVM.">
+              Peak Execution Memory OnHeap / OffHeap</span></th>
+          <th>
+            <span data-toggle="tooltip" data-placement="top"
+            title="Peak storage onHeap / OffHeap memory used by JVM.">
```

Comment:
I think we should expand this to be  "Peak onHeap/ OffHeap memory used for storage of data like RDD partitions cached in memory"